Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM busybox
+
+# The commit sha we wish to ship to prod
+ENV SHA master
+
+RUN apk --update add curl
+
+RUN curl -fsSL https://github.com/mozilla-conduit/phabricator-extensions/archive/${SHA}.tar.gz -o extensions.tar.gz \
```

Comment:
Yeah just use

COPY . /app/phabricator/src/extensions/

The docker hub images will always have the latest code since they are rebuilt for any git commits.